### ISSUE TITLE
Using removed Babel 5 option

### ISSUE THREAD

**Issue by alecguintu** [MicheleBertoli/react-gmaps#42]
I tried to use this today but I get an error on my webpack. What happening? I can add info I don't know what thoguh.

```
    ERROR in ./~/react-gmaps/dist/index.js
    Module build failed: ReferenceError: [BABEL] .../node_modules/react-gmaps/dist/index.js: Using removed Babel 5 option: .../node_modules/react-gmaps/.babelrc.stage - Check out the corresponding stage-x presets http://babeljs.io/docs/plugins/#presets
        at Logger.error (.../node_modules/babel-core/lib/transformation/file/logger.js:39:11)
        at OptionManager.mergeOptions (.../node_modules/babel-core/lib/transformation/file/options/option-manager.js:265:20)
        at OptionManager.addConfig (.../node_modules/babel-core/lib/transformation/file/options/option-manager.js:207:10)
        at OptionManager.findConfigs (.../node_modules/babel-core/lib/transformation/file/options/option-manager.js:413:16)
        at OptionManager.init (.../node_modules/babel-core/lib/transformation/file/options/option-manager.js:461:12)
        at File.initOptions (.../node_modules/babel-core/lib/transformation/file/index.js:194:75)
        at new File (.../node_modules/babel-core/lib/transformation/file/index.js:123:22)
        at Pipeline.transform (.../node_modules/babel-core/lib/transformation/pipeline.js:45:16)
        at transpile (.../node_modules/babel-loader/index.js:14:22)
        at Object.module.exports (.../node_modules/babel-loader/index.js:88:12)
     @ ./src/components/list-page.js 22:18-40
```

**Comment by MicheleBertoli** [comment]
hey @alecguintu, can you please tell me which version of `react-gmaps` are you using?

**Comment by MicheleBertoli** [comment]
Thanks @alecguintu, it should be fixed in [1.4.4](https://github.com/MicheleBertoli/react-gmaps/releases/tag/1.4.4).
